### PR TITLE
Styled the default scrollbar

### LINKED ISSUE #26: [Feature Request]: style default scrollbar
### Is your feature request related to a problem? Please describe.

Currently, the default scrollbar looks ugly.

### Describe the solution you'd like

style the scrollbar under global styling.

### Describe alternatives you've considered

_No response_

### Developer Help

_No response_

### PR DESCRIPTION
**Pull request by Doraemon012**
### Changes

- Fixes #26 
- Styled the default scrollbar as per theme of the project

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import Navbar from './components/Navbar';
 import Footer from './components/Footer';
 import BusSchedule from './pages/BusSchedule';
 import BusDetails from './pages/BusDetails';
+import './scroll.css';
```
Comment (prajjwalkapoor) [inline frontend/src/App.tsx:11]: No need for this. Because this file doesn't even exist, you can remove this.

**Review by prajjwalkapoor — CHANGES_REQUESTED** [review]
Great Work! @Doraemon012.  Just do the one suggested change and we are good to go.

**Review by Doraemon012 — COMMENTED** [review]
Sir, I removed the extra import statement.

**Review by prajjwalkapoor — APPROVED** [review]
LGTM.

**Pull request merged**